File: Modul-6/PRAK605/PRAK605-2310817210010-Harry-Pratama-Yunus.py
n=int(input())
matriks_a=[]
matriks_b=[]
print("Matriks A")
for i in range(n):
    baris =[int(x) for x in input().split()]
    matriks_a.append(baris)
print("Matriks B")
for i in range(n):
    baris =[int(x) for x in input().split()]
    matriks_b.append(baris)
print("Matriks AXB")
hasil=0
for i in range(n):
    for j in range(n):
        for k in range(n):
            hasil+=matriks_a[i][k]*matriks_b[k][j]
        print(hasil,end=" ")
        hasil=0
    print()

File: Modul-6/PRAK605/PRAK605-2310817210010-Harry-Pratama-Yunus.c
#include<stdio.h>

int main(){
    int n;
    scanf("%d",&n);
    int matriks_a[n][n],matriks_b[n][n];
    printf("Matriks A\n");
    for(int i=1;i<=n;i++){
        for(int j=1;j<=n;j++){
            scanf("%d",&matriks_a[i][j]);
        }
    }
    printf("Matriks B\n");
    for(int i=1;i<=n;i++){
        for(int j=1;j<=n;j++){
            scanf("%d",&matriks_b[i][j]);
        }
    }
    printf("Matriks AXB\n");
    int hasil=0;
    for(int i=1;i<=n;i++){
      for(int j=1;j<=n;j++){
        for(int k=1;k<=n;k++){
          hasil+=matriks_a[i][k]*matriks_b[k][j];
        }
        printf("%d ",hasil);
        hasil=0;
      }
      printf("\n");
    }
}